roles/install_yamls/tasks/zuul_set_operators_repo.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# When using CI (Zuul) to deploy OpenStack control plane with install_yamls,
```

Comment:
applies to almost everything that is deployed in install_yamls, the control plane was just an example. But yeah, it would be more clear if I mention everything here instead